src/SamplesApp/SamplesApp.UITests/TestFramework/ImageAssert.ExpectedPixels.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Drawing;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using SamplesApp.UITests._Utils;
+
+namespace SamplesApp.UITests.TestFramework
+{
+	public struct ExpectedPixels
+	{
+		#region Fluent declaration
+		public static ExpectedPixels At(string name, float x, float y)
+			=> new ExpectedPixels(name, new Point((int) x, (int)y), default, new Color[0, 0]);
+
+		public static ExpectedPixels At(float x, float y, [CallerLineNumber] int line = -1)
+			=> new ExpectedPixels($"at line: {line}", new Point((int)x, (int)y), default, new Color[0, 0]);
+
+		public static ExpectedPixels At(Point location, [CallerLineNumber] int line = -1)
+			=> new ExpectedPixels($"at line: {line}", location, default, new Color[0, 0]);
+
+		public static ExpectedPixels At(string name, Point location)
+			=> new ExpectedPixels(name, location, default, new Color[0, 0]);
+
+		public ExpectedPixels Named(string name)
+			=> new ExpectedPixels(name, Location, SourceLocation, Values, Tolerance);
+
+		public ExpectedPixels Pixels(string[,] pixels)
+		{
+			var colors = new Color[pixels.GetLength(0), pixels.GetLength(1)];
+			for (var py = 0; py < pixels.GetLength(0); py++)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'for' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5363624)